pkgs/tools/package-management/nix-prefetch/default.nix

```diff
@@ -7,27 +7,19 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "nix-prefetch";
-  version = "0.4.0";
+  version = "0.4.1";
 
   src = fetchFromGitHub {
     owner = "msteen";
     repo = "nix-prefetch";
     rev = version;
-    sha256 = "11792677zyi06jw641xi9aywwgh9002b8406w6qids212c14va6n";
+    sha256 = "0ddpj0qykm8dvjndl9r187q5gwdir3silaik7qrwx9akmah4wi6b";
     # the stat call has to be in a subshell or we get the current date
     extraPostFetch = ''
       echo $(stat -c %Y $out) > $out/.timestamp
     '';
   };
 
-  patches = [
-    # Fix input_type regex for macOS systems: https://github.com/msteen/nix-prefetch/pull/24
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
I missed that, thanks! It is now removed.